src/Sarif/Visitors/RebaseUriVisitor.cs

```diff
@@ -61,23 +63,27 @@ public RebaseUriVisitor(string baseName, bool rebaseRelativeUris, Uri baseUri)
             _rebaseRelativeUris = rebaseRelativeUris;
         }
 
-        public override PhysicalLocation VisitPhysicalLocation(PhysicalLocation node)
```

Comment:
>VisitPhysicalLocation [](start = 41, length = 21)

I take it that the reason for this change is that you want to visit every `FileLocation`, and not every `FileLocation` occurs within a `PhysicalLocation`.